terraform/main.tf

```diff
@@ -36,6 +36,77 @@ provider "aws" {
   profile                 = "aeternity"
 }
 
+module "aws_deploy-main-ap-southeast-1" {
+  source            = "modules/cloud/aws/deploy"
+  env               = "main"
+  bootstrap_version = "PT-161013282_mainnet"
+  vault_role        = "epoch-node"
+  vault_addr        = "${var.vault_addr}"
+
+  static_nodes = 1
+  spot_nodes   = 2
+
+  spot_price    = "0.15"
+  instance_type = "c5.xlarge"
+  ami_name      = "epoch-ubuntu-16.04-v1542910070"
+
+  epoch = {
+    package = "https://s3.eu-central-1.amazonaws.com/aeternity-epoch-builds/epoch-latest-ubuntu-x86_64.tar.gz"
```

Comment:
Can you please remind me which version of the application this is meant to be?

Reply:
This is latest master, will update it to fixed version number once RC2 is released